Helper/Cart.php

```diff
@@ -375,11 +375,12 @@ private function getSignResponse($signRequest)
      * Get the hints data for checkout
      *
      * @param string $cartReference            (immutable) quote id
+     * @param string $checkoutType             'cart' | 'admin' Default to `admin`
      *
      * @return array
      * @throws NoSuchEntityException
      */
-    public function getHints($cartReference = null)
+    public function getHints($cartReference = null, $checkoutType = 'admin')
```

Comment:
Perhaps the default $checkoutType should be 'cart'

Reply:
@danac-gs if the default value is `cart`, then https://github.com/BoltApp/bolt-magento2/blob/develop/Controller/Adminhtml/Cart/Data.php#L109 would be $hints = $this->cartHelper->getHints(null, 'admin');  I think there is no need to send default value for the first variable, so I choose to set the second variable default to `admin`

Reply:
@dumega makes sense